climada/util/test/test_lines_polys_handler.py

```diff
@@ -144,6 +144,25 @@ def test_point_exposure_from_polygons(self):
             ])
         np.testing.assert_allclose(exp_pnt.gdf.latitude, lat)
 
+        #projected crs, to_meters=TRUE, FIX, dissag_val
+        res = 20000
+        EXP_POLY_PROJ = Exposures(GDF_POLY.to_crs(epsg=28992))
+        exp_pnt = u_lp.exp_geom_to_pnt(
+            EXP_POLY_PROJ, res=res, to_meters=True,
+            disagg_met=u_lp.DisaggMethod.FIX, disagg_val=res**2
+            )
+        self.check_unchanged_exp(EXP_POLY_PROJ, exp_pnt)
+        val = res**2
+        self.assertEqual(np.unique(exp_pnt.gdf.value)[0], val)
+        lat = np.array([574891.12225222, 547411.67251407, 499789.43052324,
+                        460177.36473906, 364182.25061015, 369862.57549558,
+                        394014.84676059, 444749.18166595, 514229.75466288,
+                        568740.1294081 , 507005.3662343 , 458457.48789088])
+        np.testing.assert_allclose(
+            exp_pnt.gdf.groupby(level=[0])['latitude'].nth(0).values,
+            lat, atol=100000)
```

Comment:
You're right, strictly speaking one should test that `_interp_one_poly_m` does not get called and `_interp_one_poly` is called instead in this example. But that surpassed my leisurely testing skills :D 

I re-did this test now, just checking that the metre-based projection wasnt changed by the to_metres command (which indirectly checks this).

Not sure how mocking works, but could be an option for (future?) re-writes of the lines_polys_tests? It still has untested functions, like the ones you mentioned..

Reply:
Anyways, this is not what's happening apparently (jenkins complains that the crs was overwritten to epsg:4326) - a behaviour which i cannot reproduce locally, where all the tests pass. will try to find out how this is possible

Reply:
@Evelyn-M done!

Reply:
> jenkins complains that the crs was overwritten to epsg:4326

@Evelyn-M This is probably because the code uses `pd.concat` to concatenate an empty (EPSG:4326) GeoDataFrame with the created one, and places it into a new GeoDataFrame. Either the empty frame overwrites the CRS of the created one, or the CRS is dropped along the way by `pd.concat`. See https://github.com/CLIMADA-project/climada_python/blob/85650dc5925615177db6f82897e50a37574ee878/climada/util/lines_polys_handler.py#L525 and following lines

Reply:
The bug was not in that line, but in the function swapping the geometry column. It happens that when one renames the geometry column, the crs information is lost.

Fixed in 87f2e8ed4ea379ecea9beef5a9cbe775f4c0748d

Reply:
Additional info: line 525, which initializes an empty geodataframe does not fix a crs. When an empty geodataframe is created, not geometry column is defined, and thus no crs.

Reply:
great, nice work. thanks both @chahank , @peanutfun !
I guess we can merge and close this?